modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/ClassicEditor.js

```diff
@@ -78,6 +84,12 @@ const ClassicEditor = forwardRef(
 							},
 							noSnapshot: true,
 						});
+						if (required) {
+							editor.document.$.activeElement.setAttribute(
+								'aria-required',
+								true
+							);
```

Comment:
This looks flaky. There is no way to know for sure what element will be active when `onInstanceReady` triggers. You can use `editor.editable()` to get a reference to the editable area, if that is what you need.

Reply:
Hi @markocikos,
I tried it:
`editor.editable().setAttribute( 'aria-required', true )`
but it is not reflected in the DOM.
Could you help with it, please?

Reply:
@vngoliveira I'd hold off doing anything until we get a guideline from PE team in the JIRA ticket.